yak/src/yak_server.cpp

```diff
@@ -7,7 +7,7 @@ yak::FusionServer::FusionServer(const kfusion::KinFuParams& params, const Eigen:
   , last_camera_pose_(Eigen::Affine3f::Identity())
 {
   // Debug displays
-  cv::namedWindow("output", CV_WINDOW_NORMAL);
+  cv::namedWindow("output", cv::WINDOW_NORMAL);
```

Comment:
Is `cv::WINDOW_NORMAL` available in both OpenCV 2 and 3?

Reply:
Looks like this is actually a question of if it's available in OpenCV 2, 3, **and 4**, since it looks like the [Canonical version of OpenCV for Focal is 4.2](https://packages.ubuntu.com/search?keywords=opencv).

Reply:
`WINDOW_NORMAL` should be a valid flag for OpenCV [2.4](https://docs.opencv.org/2.4/modules/highgui/doc/user_interface.html#namedwindow), [3.2](https://docs.opencv.org/3.2.0/d7/dfc/group__highgui.html#gabf7d2c5625bc59ac130287f925557ac3), and [4.2](https://docs.opencv.org/4.2.0/d7/dfc/group__highgui.html#gabf7d2c5625bc59ac130287f925557ac3), so this is the right change to make.

Reply:
> `WINDOW_NORMAL` is a valid flag for OpenCV [2.4](https://docs.opencv.org/2.4/modules/highgui/doc/user_interface.html#namedwindow), [3.2](https://docs.opencv.org/3.2.0/d7/dfc/group__highgui.html#gabf7d2c5625bc59ac130287f925557ac3), and [4.2](https://docs.opencv.org/4.2.0/d7/dfc/group__highgui.html#gabf7d2c5625bc59ac130287f925557ac3), so this is the right change to make.

cv::WINDOW_NORMAL compiles on focal